src/components/article-card/article-card.jsx

```diff
@@ -12,6 +17,7 @@ function ArticleCard(props){
         <p>{article.comments}</p>
         <p>{article.created_at}</p>
```

Comment:
can we make this more readable? this is currently a date time stamp which makes it hard to read!